exercises/shared/.docs/tests.md

```diff
@@ -1,14 +1,109 @@
 # Running tests
 
-Execute the tests with:
+Tests are written in test files and placed in the `test` directory and implemented
+as:
+
+```elixir
+test "short test description" do
+  ... test implementation ...
+end
+```
+
+All tests may be executed with:
 
 ```bash
 $ mix test
 ```
 
-## Pending tests
+Useful top-level references include:
+
+* Elixir's unit test library: https://hexdocs.pm/ex_unit/ExUnit.html
+* Elixir's test execution tool: https://hexdocs.pm/mix/Mix.Tasks.Test.html
+
+`ExUnit` and `mix` provides quite a few methods of grouping, tagging & executing
+tests and various methods of controlling test execution, much of which is summarized
+below.
 
-In the test suites, all but the first test have been skipped.
+### Methods for running specific tests
+
+`mix` has a variety of methods of running specific and specific groups of tests.
+
+Below is just a summary of some very useful methods; full documentation is here:
+https://hexdocs.pm/mix/Mix.Tasks.Test.html
+
+##### Running tests in a specific file
+
+All tests in a single file may be executed via `mix` by specifying the file:
+
+```bash
+$ mix test test/<FILE>.ex
```

Comment:
Test files are always `.exs` files (also applies to a few other code blocks below)`

Reply:
will do!

Reply:
pushed